numpy/lib/recfunctions.py

```diff
@@ -153,9 +157,13 @@ def get_names_flat(adtype):
     --------
     >>> from numpy.lib import recfunctions as rfn
     >>> rfn.get_names_flat(np.empty((1,), dtype=int)) is None
-    True
+    Traceback (most recent call last):
+        ...
+    AttributeError: 'numpy.ndarray' object has no attribute 'names'
```

Comment:
This looks like a bug - perhaps just open it later

Reply:
see #12485 -- I'm inclined to leave the "truth" (a traceback) until this is fixed / cleared up, but can add a skip otherwise.